tests/e2e/standard/contacts/create-places-contacts.wdio-spec.js

```diff
@@ -0,0 +1,44 @@
+const loginPage = require('../../../page-objects/default/login/login.wdio.page');
+const commonPage = require('../../../page-objects/default/common/common.wdio.page');
+const contactPage = require('../../../page-objects/standard/contacts/contacts.wdio.page');
+
+const PLACE_TYPES = { CLINIC: 'clinic', HEALTH_CENTER: 'health_center', DISTRICT: 'district_hospital' };
+const CONTACT_NAME_SUFFIX = ' Contact';
+
+describe('Creating contacts with standard config. ', () => {
+  const districtName = 'District test';
+  const healthCenterName = 'Health Center test';
+  const clinicName = 'Clinic test';
+
+  before(async () => {
+    await loginPage.cookieLogin();
+    await commonPage.waitForPageLoaded();
+  });
+
+  it('should create a new district hospital', async () => {
+    await commonPage.goToPeople();
+    await contactPage.addPlace(PLACE_TYPES.DISTRICT, districtName, districtName + CONTACT_NAME_SUFFIX);
```

Comment:
It looks like `districtName` is used as constant, then it'd be better to use the same naming style as  `CONTACT_NAME_SUFFIX`.
For example: DISTRICT_NAME

Optionally, you can use actual names like "Blue Rose Hospital" and "Ana Maria Gonzales de la Trinidad". 
For tests cases it's okay to repeat the text in the test data and in the expects, since most cases helps to easily see what is expected instead of searching for the constant value.... 

Either way is fine